Dockerfile

```diff
@@ -8,5 +8,28 @@
 # TARGETARCH
 FROM docker.io/library/alpine:3.18.2@sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1
 
-RUN apk add --no-cache curl iputils bind-tools tcpdump
+RUN apk add --no-cache curl iputils bind-tools tcpdump tar
+
+# INSTALL dnsperf and its dependencies
+RUN apk update \
+    && apk del libressl-dev \
+    && apk add openssl-dev ldns-dev ck-dev\
+    && apk add pkgconfig build-base gcc libtool
+WORKDIR /usr/bin
```

Comment:
any reason to use this workdir? IMO we can still copy the compiled binary after?